scripts/validate_excel.py

```
@@ -75,7 +75,8 @@ def is_covered(slot_start, slot_end):
             current_date_obj = datetime.strptime(
                 date_str[4:].strip(), "%d/%m/%y"
             ).date()
-        except Exception as e:
+        except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)